docs/custom_conf.py

```diff
@@ -16,8 +21,8 @@
 
 def _compute_navigation_tree(context):
     tree_html = _old_compute_navigation_tree(context)
-    if not tree_html and context.get("toc"):
-        tree_html = furo.navigation.get_navigation_tree(context["toc"])
+    if not tree_html and context.get('toc'):
```

Comment:
Do we need to add this to automatic Ruffing?

Reply:
Yeah, that's probably a good idea.